assembly/hdp2.6.5/build/var/lib/ambari-server/resources/common-services/SPLICEMACHINE/2.5.1/service_advisor.py

```diff
@@ -0,0 +1,240 @@
+        message = "Splice Machine requires this property to be set to the recommended value of " + desired_value
+        validationItems.append({"config-name": property, "item": self.getWarnItem(message)})
+    return self.toConfigurationValidationProblems(validationItems, "core-site")
+
+  def validateHDFSSiteConfigurations(self, properties, recommendedDefaults, configurations, services, hosts):
+      print "validateHDFSSiteConfigurations"
+      hdfs_site = properties
+      validationItems = []
+      for property, desired_value in self.getHDFSSiteDesiredValues().iteritems():
+          if property not in hdfs_site or hdfs_site[property] != desired_value:
+              message = "Splice Machine requires this property to be set to the recommended value of " + desired_value
+              validationItems.append({"config-name": property, "item": self.getWarnItem(message)})
+      return self.toConfigurationValidationProblems(validationItems, "hdfs-site")
+
+  def validateHBaseSiteConfigurations(self, properties, recommendedDefaults, configurations, services, hosts):
+      print "validateHBaseSiteConfigurations"
+      hbase_site = properties
+      validationItems = []
+      for property, desired_value in self.getHBaseSiteDesiredValues().iteritems():
+          print "->" + property + ":" + desired_value + ":" + hbase_site[property]
+          if property not in hbase_site or hbase_site[property] != desired_value:
+              message = "Splice Machine requires this property to be set to the recommended value of " + desired_value
+              validationItems.append({"config-name": property, "item": self.getWarnItem(message)})
+      return self.toConfigurationValidationProblems(validationItems, "hbase-site")
+
+  def getCoreSiteDesiredValues(self):
+    core_site_desired_values = {
+        "ipc.server.listen.queue.size" : "3300"
+    }
+    return core_site_desired_values
+
+  def getHDFSSiteDesiredValues(self):
+    hdfs_site_desired_values = {
+        "dfs.datanode.handler.count" : "20",
+        "dfs.client.read.shortcircuit.buffer.size" : "131072",
+
+    }
+    return hdfs_site_desired_values
+
+  def getHBaseSiteDesiredValues(self):
+    hbase_site_desired_values = {
+        "hbase.coprocessor.master.classes" : "com.splicemachine.hbase.SpliceMasterObserver",
+        "hbase.regionserver.global.memstore.size" : "0.25",
+        "hfile.block.cache.size" : "0.25",
+        "hbase.regionserver.handler.count" : "200",
+        "hbase.client.scanner.caching" : "1000",
+        "hbase.hstore.blockingStoreFiles" : "20",
+        "hbase.hstore.compactionThreshold" : "5",
+        "hbase.balancer.period" : "60000",
+        "hbase.client.ipc.pool.size" : "10",
+        "hbase.client.max.perregion.tasks" : "100",
+        "hbase.coprocessor.regionserver.classes" : "com.splicemachine.hbase.RegionServerLifecycleObserver",
+        "hbase.hstore.compaction.max.size" : "260046848",
+        "hbase.hstore.compaction.min.size" : "16777216",
```

Comment:
DB-7656 recently made changes to these default values.  For example, hbase.hstore.compaction.min.size was changed to 136314880 and hbase.hstore.compaction.max.size was removed entirely.  Please review DB-7656 and apply the same changes to this pull request.